XenModel/XenAPI/VM.cs

```diff
@@ -3590,7 +3590,7 @@ public static void set_protection_policy(Session session, string _vm, string _va
         /// <param name="_value">The value</param>
         public static void set_snapshot_schedule(Session session, string _vm, string _value)
         {
-            session.proxy.vm_set_snapshot_schedule(session.uuid, (_vm != null) ? _vm : "", (_value != null) ? _value : "").parse();
+            session.proxy.vm_set_snapshot_schedule(session.uuid, (_vm != null) ? _vm : "", (_value != null) ? _value : Helper.NullOpaqueRef).parse();

```

Comment:
This file is auto generated from the xen-api, and we shouldn't change it, except by updating it with the generated one.
If we need to pass "OpaqueRef:NULL" to XAPI instead of the empty string, then we should make sure we don't call this function with _value==null (and explicitly use "OpaqueRef:NULL" when we want to unassign a VM from a policy)